include/envoy/stats/histogram.h

```diff
@@ -105,16 +95,6 @@ class ParentHistogram : public virtual Histogram {
    * Returns the cumulative histogram summary statistics.
    */
   virtual const HistogramStatistics& cumulativeStatistics() const PURE;
-
```

Comment:
`quantileSummary` was only used in admin.cc, and `bucketSummary` was unused entirely.